src/Controller/EnMarche/TerritorialCouncil/PoliticalCommitteeController.php

```diff
@@ -92,6 +96,53 @@ public function deleteFeedItemAction(Request $request, PoliticalCommitteeFeedIte
     {
         $this->deleteFeedItem($request, $feedItem, FeedItemTypeEnum::POLITICAL_COMMITTEE);
 
-        return $this->redirectToRoute('app_political_committee_messages');
+        return $this->redirectToRoute('app_political_committee_index');
+    }
+
+    /**
+     * @Route("/proces-verbaux", name="official_report_list", methods={"GET"})
+     */
+    public function listOfficialReportsAction(
+        UserInterface $adherent,
+        OfficialReportRepository $reportRepository
+    ): Response {
+        $reports = $reportRepository->getReportsForPoliticalCommittee(
+            $adherent->getPoliticalCommitteeMembership()->getPoliticalCommittee()
+        );
+
+        return $this->render('territorial_council/political_committee/official_reports.html.twig', [
+            'reports' => $reports,
+        ]);
+    }
+
+    /**
+     * @Route("/proces-verbaux/{uuid}/telecharger", name="official_report_download", methods={"GET"})
+     */
+    public function downloadReportAction(
+        Request $request,
+        OfficialReport $officialReport,
+        FilesystemInterface $storage
+    ): Response {
+        $lastDocument = $officialReport->getLastDocument();
```

Comment:
do we need to check here that the OfficialReport has the same politicalCommittee that the membership?